src/aiida/manage/configuration/profile.py

```diff
@@ -29,41 +31,44 @@
 class Profile:
     """Class that models a profile as it is stored in the configuration file of an AiiDA instance."""
 
-    KEY_UUID = 'PROFILE_UUID'
-    KEY_DEFAULT_USER_EMAIL = 'default_user_email'
-    KEY_STORAGE = 'storage'
-    KEY_PROCESS = 'process_control'
-    KEY_STORAGE_BACKEND = 'backend'
-    KEY_STORAGE_CONFIG = 'config'
-    KEY_PROCESS_BACKEND = 'backend'
-    KEY_PROCESS_CONFIG = 'config'
-    KEY_OPTIONS = 'options'
-    KEY_TEST_PROFILE = 'test_profile'
+    KEY_UUID: str = 'PROFILE_UUID'
+    KEY_DEFAULT_USER_EMAIL: str = 'default_user_email'
+    KEY_STORAGE: str = 'storage'
```

Comment:
mypy not complain, again, it is a suggestion from pyright. For the class attributes, if the class is not annotate with @final, better to add type to avoid being wrongly override.

Reply:
Too keep the diff small, I would prefer to not do unrelated changes in this PR, since it should be reviewed carefully given that it touches core functionality.